docker/build-sudo-images.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Build fast OS docker images that run slower steps ahead of time, to save time running tests
+
+cd "$(dirname "$0")"
+sed -i .orig -e "s#@@MYUSERID@@#`id -u`#g" sudo-*/Dockerfile
+docker build -t sudo-debian:wheezy sudo-debian
+docker build -t sudo-centos:6 sudo-centos6
+docker build -t sudo-centos:7 sudo-centos7
+docker build -t sudo-opensuse:13.2 sudo-opensuse
+docker build -t sudo-ubuntu:14.04 sudo-ubuntu14
```

Comment:
@kwhetstone I've included documentation to better explain the use case here, but this is just a helper script to build the test images. 

There's not much reason to only build a subset of images: the Docker cache makes all builds after the first extremely fast (a few seconds), and if you want to only build part of them, you can build manually or modify the script itself easily.